### ISSUE TITLE
wifinina: http-get example fails due to too small of buffer

### ISSUE THREAD

**Issue by scottfeldman** [tinygo-org/drivers#470]
Running examples/wifinina/http-get example on Arduino Nano33 IoT.  Example fails with this error:
```
slice out of range : use http.SetBuf() to change the allocation to 1036 bytes or more 
```
Fix is to increase buffer size:
```
--- a/examples/wifinina/http-get/main.go
+++ b/examples/wifinina/http-get/main.go
@@ -44,7 +44,7 @@ var (
        adaptor *wifinina.Device
 )
 
-var buf [0x400]byte
+var buf [0x500]byte
```
I'll prepare a PR if the change is acceptable...

$ tinygo version
tinygo version 0.25.0 linux/arm (using go version go1.19.1 and LLVM version 14.0.0)

**Comment by scottfeldman** [comment]
Just a note for now: this issue is fixed with the netdev branch.

**Comment by scottfeldman** [comment]
Fixed by #537.

**Comment by deadprogram** [comment]
Closing as completed in the most recent release. Thank you!
